docs/using_deis/config-application.rst

```diff
@@ -91,6 +91,28 @@ appname to the old one:
     an application, however you can work around this by pointing the @ record to the
     address of the load balancer (if any).
 
+Custom Health Checks
+--------------------
+
+By default, Deis checks if the container is running, performing no health checks. You can add a
+healthcheck by adding a healthcheck URL, the initial delay and the timeout value by setting custom
+config:
```

Comment:
This could be phrased more tersely:
> By default, Deis only checks that a container is running. You can add a healthcheck by configuring a URL, initial delay, and timeout value: